packages/api/src/schema/api-schema.yaml

```
@@ -0,0 +1,3102 @@
+                errorMessage:
+                  type: string
+                  description: Error message if the last storage changed failed.
+                tasks:
+                  type: object
+                  additionalProperties: false
+                  properties:
+                    pending:
+                      type: string
+                      description: >-
+                        ID of any currently running task that is exporting this
+                        asset to IPFS.
+                    last:
+                      type: string
+                      description: >-
+                        ID of the last task to run successfully, that created
+                        the currently saved data.
+                    failed:
+                      type: string
+                      description: ID of the last task to fail execution.
+        status:
+          type: object
+          additionalProperties: false
+          required:
+            - phase
+            - updatedAt
+          description: Status of the asset
+          properties:
+            phase:
+              type: string
+              description: Phase of the asset
+              enum:
+                - uploading
+                - waiting
+                - processing
+                - ready
+                - failed
+            updatedAt:
+              type: number
+              description:
+                Timestamp (in milliseconds) at which the asset was last updated
+              example: 1587667174725
+            progress:
+              type: number
+              description: Current progress of the task creating this asset.
+            errorMessage:
+              type: string
+              description: Error message if the asset creation failed.
+        name:
+          type: string
+          description: >-
+            Name of the asset. This is not necessarily the filename, can be a
+            custom name or title
+          example: filename.mp4
+        createdAt:
+          type: number
+          description: Timestamp (in milliseconds) at which asset was created
+          example: 1587667174725
+        size:
+          type: number
+          description: Size of the asset in bytes
+          example: 84934509
+        hash:
+          type: array
+          description: Hash of the asset
+          items:
+            type: object
+            additionalProperties: false
+            properties:
+              hash:
+                type: string
+                description: Hash of the asset
+                example: >-
+                  9b560b28b85378a5004117539196ab24e21bbd75b0e9eb1a8bc7c5fd80dc5b57
+              algorithm:
+                type: string
+                description: Hash algorithm used to compute the hash
+                example: sha256
+        videoSpec:
+          type: object
+          additionalProperties: false
+          description: Video metadata
```

```suggestion
          description:
            Specific video metadata about the asset. Most of these fields are only returned if a 
            `?details=true` query-string is present on the request.
```
(not sure if the indentation is right)